.github/workflows/test.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Load Codecov token from 1Password
         id: load-codecov-token
         # Codecov needs a token to be run on the default branch (which happens in preDeploy.yml after a PR is merged to main)
-        if: ${{ github.event_name == 'workflow_call' && github.ref == 'refs/heads/main' }}
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Comment:
Interesting! I didn't realize that `workflow_call` inherited the parent `event_name`.

Reply:
Did you test this somewhere, or how did you figure this out?